apps/web/components/shared/learning-paths.tsx

```
@@ -0,0 +1,103 @@
+				Choose Your Learning{' '}
+				<span className="bg-gradient-to-l from-secondary to-primary bg-clip-text text-transparent">
+					Journey
+				</span>
+			</h2>
+			``
+			{/* Learning Path Cards */}
+			<div className="grid grid-cols-1 md:grid-cols-2 gap-6 mt-6">
+				{learningPaths.map((path, index) => {
+					const IconComponent = iconMap[path.icon]
+					return (
+						<Card key={index} className="p-4 shadow-lg flex items-start">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)